packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -5,13 +5,10 @@ import {
   initializeWithHeadless,
 } from 'c/quanticHeadlessLoader';
 
+import { keys } from 'c/quanticUtils';
```

Comment:
prettier difference again :$

Reply:
Does it really matter? Until we have a commit hook or a prettierjs config file we'll be getting those ;)